src/Core/Resolvers/MySqlQueryExecutor.cs

```diff
@@ -136,15 +135,12 @@ private bool IsDefaultAccessTokenValid()
         {
             try
             {
-                _defaultAccessToken =
-                    await AzureCredential.GetTokenAsync(
-                        new TokenRequestContext(new[] { DATABASE_SCOPE }));
+                _defaultAccessToken = await AzureCredential.GetTokenAsync(new TokenRequestContext(new[] { DATABASE_SCOPE }));
             }
             catch (CredentialUnavailableException ex)
             {
-                QueryExecutorLogger.LogWarning($"{HttpContextExtensions.GetLoggerCorrelationId(HttpContextAccessor.HttpContext)}" +
-                    $"Attempt to retrieve a managed identity access token using DefaultAzureCredential" +
-                    $" failed due to: \n{ex}");
+                string correlationId = HttpContextExtensions.GetLoggerCorrelationId(HttpContextAccessor.HttpContext);
+                QueryExecutorLogger.LogWarning("{CorrelationId} Failed to retrieve a managed identity access token using DefaultAzureCredential: {message}", correlationId, ex.Message);
```

Comment:
```suggestion
                QueryExecutorLogger.LogWarning("{correlationId} Failed to retrieve a managed identity access token using DefaultAzureCredential: {message}", correlationId, ex.Message);
```

Just being consistent with naming

Reply:
updated